### PR TITLE
8.3 XS82ECU1063

### PR DESCRIPTION
**Pull request by gduperrey**
Koji build (scratch) to v8.3-incoming: : https://koji.xcp-ng.org/taskinfo?taskID=5435

### CODE REVIEW

File: SPECS/curl.spec [hunk 1]
```diff
@@ -41,17 +41,12 @@ BuildRequires: openldap-devel
 BuildRequires: openssh-clients
 BuildRequires: openssh-server
 BuildRequires: openssl-devel
-BuildRequires: perl-interpreter
+BuildRequires: perl
```
Comment (stormi) [inline SPECS/curl.spec]: Not necessary. `perl-interpreter` exists in 8.3.

File: SPECS/curl.spec [hunk 2]
```diff
@@ -322,7 +308,9 @@ rm -f ${RPM_BUILD_ROOT}%{_libdir}/libcurl.la
 # should be fixed in next release https://github.com/curl/curl/pull/12843
 rm -f ${RPM_BUILD_ROOT}%{_mandir}/man1/mk-ca-bundle.1*
 
-%ldconfig_scriptlets -n libcurl
+%post -n libcurl -p /sbin/ldconfig
+
+%postun -n libcurl -p /sbin/ldconfig
```
Comment (stormi) [inline SPECS/curl.spec]: The `%ldconfig_scriptlets` macro is actually available in our build root in 8.3, so we can keep using it here.

**Review by stormi — APPROVED** [review]

**Review by bleader — APPROVED** [review]

**Comment by gduperrey** [comment]
Koji build (scratch) to v8.3-incoming: https://koji.xcp-ng.org/taskinfo?taskID=54675

**Comment by gduperrey** [comment]
Koji build (v8.3-incoming): https://koji.xcp-ng.org/taskinfo?taskID=54680

**Pull request merged**